source/reference/minio-server/minio-server.rst

```diff
@@ -419,6 +420,14 @@ Key Management Service and Encryption
    - The encryption key for Server-Side Encryption with
      :ref:`SSE-S3 <minio-encryption-sse-s3>`.
 
+.. envvar:: MINIO_KMS_KES_ENCLAVE
+
+   Use this optional environment variable to define the name of a KES enclave.
+   A KES enclave provides as an isolated space within the server for keys to allow for tenant separation.
```

Comment:
I'd like to have a little more detail here - how exactly does this isolation work or help? Is it similar to a namespace? Does the KES IAM help here in some way w.r.t. key isolation between clients?

cc @aead

Reply:
Added an brief overview to a glossary entry and linked to the glossary from the various platform files.